wiki-docs/docs/interfaces/erc20/childerc20-predicate.md

```diff
@@ -0,0 +1,74 @@
+The `IChildERC20Predicate` interface is part of the cross-chain bridge system, which manages the child tokens on the L2 side. This guide will explain the methods provided by the IChildERC20Predicate interface and how to use them.
+
+## Overview
+
+The `IChildERC20Predicate` interface is responsible for managing child tokens on L2. It provides:
+
+- Methods for deploying new child tokens.
+- Processing state updates received from L1.
+- Withdrawing tokens back to L1.
+
+## Initialize
+
+Before using the IChildERC20Predicate contract, it must be initialized with the required parameters:
+
+```solidity
+function initialize(
```

Comment:
This function is changed, and now it looks like:

```
function initialize(
        address newL2StateSender,
        address newStateReceiver,
        address newRootERC20Predicate,
        address newChildTokenTemplate,
        address newNativeTokenRootAddress
    ) public virtual onlySystemCall initializer {
```

Reply:
updating in upcoming commit